_posts/2019-05-03-tadpoles-on-tensorflow/tadpoles-on-tensorflow.Rmd

```diff
@@ -0,0 +1,681 @@
+---
+title: "Tadpoles on TensorFlow: Hierarchical partial pooling with tfprobability"
+description: > 
+ This post is a first introduction to MCMC modeling with tfprobability, the R interface to TensorFlow Probability (TFP). Our example is a multi-level model describing tadpole mortality, which may be known to the reader from Richard McElreath's wonderful "Statistical Rethinking".  
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2019tadpoles
+date: 05-03-2019
+categories:
+  - Probability and statistics 
+  - Introductions
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/thumb.png
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+So before we jump into the technicalities, -- yes, that's what this post is too: _Un hommage à_ McElreath who wrote one of most intriguing books on Bayesian (or should we just say - scientific?) modeling we're aware of. If you haven't read _Statistical Rethinking_, and are interested in modeling, you might definitely want to check it out. In this post, we're not going to try to re-tell the story: Our clear focus will instead be a demonstration of how to do MCMC with [tfprobability](https://rstudio.github.io/tfprobability/).
```

Comment:
I think the " -- yes, that's what this post is too" is a little hard to understand with the sentence construct. I think it's implied by the first part of the sentence, so you can probably cut it.

Reply:
Offset instead with commas

Reply:
Oh yeah it's pretty convoluted... I'm now trying to simply say 

> Before we jump into the technicalities: This post is, of course, dedicated to McElreath who 

That should be at least a bit better right?

Reply:
sorry I need to ask, what does "Offset instead with commas" refer to exactly?

Reply:
Yes, better! It means use commas around instead so "will, instead, be demonstrated" (sorry if that's not verbatim, I'm on my phone now)

Reply:
oh, got it! ... and sorry for having you do this during _staycation_ !!